nitrite.go

```diff
@@ -340,43 +378,28 @@ func Verify(data []byte, options VerifyOptions) (*Result, error) {
 
 	intermediates := x509.NewCertPool()
 
-	if !options.AllowSelfSignedCert {
-		for _, item := range doc.CABundle {
-			cert, err := x509.ParseCertificate(item)
-			if nil != err {
-				return nil, err
-			}
-
-			intermediates.AddCert(cert)
-			certificates = append(certificates, cert)
+	for _, item := range doc.CABundle {
+		cert, err := x509.ParseCertificate(item)
+		if nil != err {
+			return nil, err
 		}
-	}
-
-	roots := options.Roots
-	if nil == roots {
-		roots = defaultRoot
-	}
-	if cert.IsCA {
-		roots.AddCert(cert)
-	}
 
-	currentTime := options.CurrentTime
-	if currentTime.IsZero() {
-		currentTime = time.Now()
+		intermediates.AddCert(cert)
+		certificates = append(certificates, cert)
 	}
 
 	_, err = cert.Verify(
 		x509.VerifyOptions{
 			Intermediates: intermediates,
-			Roots:         roots,
-			CurrentTime:   currentTime,
+			Roots:         rootCert(),
+			CurrentTime:   verificationTime(doc),
```

Comment:
Use the roots and verification time from options.

BTW, this is much nice than passing the attestation timestamp as an argument, because we don't have to parse the attestation twice - once to get the timestamp and once to verify it.